src/main/java/Duke.java

```diff
@@ -1,10 +1,46 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void printHorizontalLine() {
+        String horizontalLine = "____________________________________________________________";
+        System.out.println(horizontalLine);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Greeting Message
+        printHorizontalLine();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        printHorizontalLine();
+
+        // Initialize new instance of a ToDoList object
+        TaskList list = new TaskList();
```

Comment:
It's very good to set TaskList() as another class so that can avoid Duke class being too long.